website/content/docs/platform/k8s/vso/auth/aws.mdx

```diff
@@ -0,0 +1,222 @@
+---
+layout: docs
+page_title: Vault Secrets Operator AWS Auth
+description: >-
+  The Vault Secrets Operator supports authenticating to Vault with the AWS auth method
+---
+
+# AWS auth method support
+
+The Vault Secrets Operator (VSO) supports authenticating to Vault's [AWS auth](/vault/docs/auth/aws) method. AWS credentials can be retrieved in several ways:
```

Comment:
```suggestion
The Vault Secrets Operator (VSO) supports
 [AWS authentication](/vault/docs/auth/aws) when accessing Vault. VSO
 can retrieve AWS credentials:
```

Style correction: write in active voice, avoid possessives, make it clear what takes action